doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,103 @@
+.. _User_Guide:
+
+#####################
+`ts_imsim` User Guide
+#####################
+
+The `ts_imsim` closed loop is designed to run as an executable through the command line as `imgClosedLoop.py`.
+
+.. _Installing_ts_imsim:
+
+Installing `ts_imsim`
+=====================
+
+Needed Packages
+---------------
+
+- `ts_wep <https://github.com/lsst-ts/ts_wep>`_
+- `ts_ofc <https://github.com/lsst-ts/ts_ofc>`_
+- `imSim <https://github.com/LSSTDESC/imSim>`_
+- `astroplan <https://github.com/astropy/astroplan>`_
+- `black <https://github.com/psf/black>`_ (optional)
+- `documenteer <https://github.com/lsst-sqre/documenteer>`_ (optional)
+- `plantuml <http://plantuml.com>`_ (optional)
+- `sphinxcontrib-plantuml <https://pypi.org/project/sphinxcontrib-plantuml/>`_ (optional)
+
+Install ts_imsim
+-----------------------------------------
+
+1. Install and set up required packages from list above using `eups`.
+
+2. Clone the `ts_imsim` package.
+
+3. Setup the imSim environment by `eups`:
+
+```bash
+cd $ts_imsim_directory
+setup -k -r .
+scons
+```
+
+.. _Using ts_imsim:
+
+Using `ts_imsim`
+================
+
+Running the AOS Closed Loop
+---------------------------
+
+The closed loop is available from the command line after running `scons` with the command `imgClosedLoop.py`.
+
+A sample command to run 5 iterations of the closed loop with the lsst wavefront sensors is:
+
+`imgClosedLoop.py --output cwfs_imsim_output/ --iter_num 5 --config_pointer_file ts_imsim/policy/config/lsstCamNoPertPointer.yaml --turn_off_atmosphere --turn_off_sky_background`
```

Comment:
Please use a code block as above. Otherwise, it looks like this:

<img width="660" alt="Screenshot 2023-11-07 at 9 20 18 PM" src="https://github.com/lsst-ts/ts_imsim/assets/11474902/5bed9175-0671-44b5-b4d8-e91df3ae112a">